codalab/worker/local_run/local_run_state.py

```diff
@@ -254,9 +265,11 @@ def _transition_from_PREPARING(self, run_state):
                 runtime=self.docker_runtime,
             )
             self.worker_docker_network.connect(container)
-        except docker_utils.DockerException as e:
-            run_state.info['failure_message'] = 'Cannot start Docker container: {}'.format(e)
-            return run_state._replace(stage=LocalRunStage.CLEANING_UP, info=run_state.info)
+        except Exception as e:
+            message = 'Cannot start Docker container: {}'.format(e)
```

Comment:
Don't fail the bundle because it's probably the worker's fault.

Reply:
Can you add this as a comment?

Also, it looks like it is failing the bundle, since there is a `raise`?

Reply:
A general comment is at the top of the class.